code/stable/glassbr/src/cpp/input.txt

```diff
@@ -10,7 +10,7 @@
 1.0
 # glass type
 HS
-# nominal thickness t is in {2.5, 2.7, 3.0, 4.0, 5.0, 6.0, 8.0, 10.0, 12.0, 16.0, 19.0, 22.0} (mm)
+# nominal thickness (mm)
```

Comment:
There's loss of information here too.

Reply:
I managed to add it back to the definition of nominal thickness. However, as Jason mentioned above I am not sure if we still want it to be a part of the definition.